site-config.yml

```diff
@@ -1,6 +1,3 @@
 ---
-hub: "cdcepi/FluSight-forecast-hub/"
-title: "Example Hub Dashboard Website"
-pages:
-  - "about.md"
-
+hub: "CDCgov/covid19-forecast-hub/"
+title: "COVID Hub Dashboard Website"
```

Comment:
I suggest "COVID-19 Forecast Hub Dashboard"

"Dashboard Website" is redundant.